docusaurus/docs-internals/editor-controls/multivalue-controls/iconSetter.md

```diff
@@ -0,0 +1,140 @@
+
+### Parameters
+
+| Name                | Type                                     | Default | Description                                                                                                                                                                                                                       |
+| :------------------ | :--------------------------------------- | :-----: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                | `string`                                 |    -    | The identifier of the key where the iconSetter will save your data                                                                                                                                                                |
+| `type`              | `string`                                 |    -    | Type should be `"iconSetter"` to use this control                                                                                                                                                                                 |
+| `label?`            | `string`                                 |    -    | The label displayed on the left side of the control                                                                                                                                                                               |
+| `position?`         | `number`                                 |    -    | The position of the control in toolbar                                                                                                                                                                                            |
+| `devices?`          | `"all"` \| `"desktop"` \| `"responsive"` | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled?`         | `boolean`                                | `false` | Configure the condition under which the control is disabled or enabled.                                                                                                                                                           |
+| `config.canDelete?` | `boolean`                                | `false` | Determines whether the icon can be deleted.                                                                                                                                                                                       |
+
+### Return value
+
+The return value of the iconSetter control represents the current settings of the icon, including its:<br/>
+`name` - The name of the icon being set;<br/>
```

Comment:
`name` - the name of the icon
`type` - the type of the icon
`filename` - the icon file name